eslint.config.mjs

```diff
@@ -0,0 +1,126 @@
+// @ts-check
```

Comment:
Seems unusual that you added this directive. I suppose it doesn't automatically pick up mjs files?

Also why mjs? We don't normally use this format according to the template.

Reply:
This is the recommended format from [the docs](https://typescript-eslint.io/getting-started/#step-2-configuration). And indeed the `ts-check` enforces the same checks as the other files (would not by default since it is `js`).

This is now used everywhere when `eslint` was updated to the latest version, as it became the standard.